strongbox-data-service/src/main/java/org/carlspring/strongbox/config/janusgraph/EmbeddedDbServerConfiguration.java

```diff
@@ -46,7 +48,7 @@
     public static final String PATH_STRONGBOX_DB = "META-INF/org/carlsparing/strongbox/db";
 
     @Bean
-    EmbeddedDbServer embeddedDbServer(CassandraEmbeddedConfiguration cassandraConfiguration,
+    JanusGraphServer embeddedDbServer(CassandraEmbeddedConfiguration cassandraConfiguration,
```

Comment:
Shouldn't this rather be `remoteDbServer`?

Reply:
This name is for consistency between all the `...ServerConfiguration` classes. It will be better to have same bean name in all configurations.